blocks/edit/prose/index.js

```diff
@@ -205,7 +206,7 @@ export default function initProse({ path, permissions }) {
         wsProvider.awareness.setLocalStateField(
           'user',
           {
-            color: generateColor(profile.email || profile.userId),
+            color: generateColor(normalizeEmail(profile.email) || profile.userId),
```

Comment:
Misread. This **_is_** missing `toLowerCase`. I would simply add it here and be done. No need to make a utility for a built in browser function.